tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -16,18 +16,22 @@ Library   OpenShiftLibrary
 *** Variables ***
 ${JUPYTERHUB_SPAWNER_HEADER_XPATH} =
```

Comment:
Are we using this Variable anywhere ? @lugi0

Reply:
if not i think we should remove this Variable

Reply:
done